python/ray/rllib/agents/ddpg/__init__.py

```diff
@@ -4,12 +4,14 @@
 
 from ray.rllib.agents.ddpg.apex import ApexDDPGTrainer
 from ray.rllib.agents.ddpg.ddpg import DDPGTrainer, DEFAULT_CONFIG
+from ray.rllib.agents.ddpg.td3 import TD3Trainer
 from ray.rllib.utils import renamed_class
 
 ApexDDPGAgent = renamed_class(ApexDDPGTrainer)
 DDPGAgent = renamed_class(DDPGTrainer)
+TD3Agent = renamed_class(TD3Trainer)
```

Comment:
nit: no need to include the *Agent def for new algorithms, since we're renaming them all to *Trainer.